sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -994,6 +1000,11 @@ Parameters:
 
 * `graph` - Graph object to execute.
 
+Exceptions:
+
+* Throws asynchronously with error code `invalid` if the handler is submitted
```

Comment:
Went for an asynchronous error here, but not convinced on it over synchronous.

Reply:
I think in this case it would be a synchronous error because we would error before the submit call has returned.